lib/link_checker/uri_checker/http_checker.rb

```diff
@@ -1,3 +1,5 @@
+require "faraday-cookie_jar"
```

Comment:
I'd have thought this gets already required as part of the gem - just wondering if it's needed